lib/material/material_gui.flow

```diff
@@ -487,21 +487,7 @@ MAvailableWidthHeight(m : Material, width : Transform<double>, height : Transfor
 
 MZoomToFit(content : Material, box : Material) -> Material {
 	cwh = makeWH();
-	bwh = makeWH();
-	scale = fselect2(cwh, bwh, FLift2(\c, b ->
-		if (c.width <= b.width && c.height <= b.height)
-			Factor(1.0, 1.0)
-		else if (c.width == 0.0 || c.height == 0.0)
-			Factor(0.0, 0.0)
-		else
-			min(b.width / c.width, b.height / c.height)
-			|> (\fc -> Factor(fc, fc))
-	));
-
-	MGroup2(
-		MScale(scale, MInspect([ISize(cwh)], content)),
-		MFixSize(MInspect([ISize(bwh)], MVisible(make(false), box)), TEmpty())
-	)
```

Comment:
@NikitaFil I'm a bit confused about this code. Is it ok to be deleted?
I don't see any difference in behaviour with and without it.

Reply:
I`ve expected from new implementation to fail in cases like `MLines2(MZoomToFit(...), MZoomToFit(...))`. 
But as it works fine both in separate testcases and lyceum usage, then ok to delete previous.